app/assets/stylesheets/course/announcements.scss

```diff
@@ -1,15 +1,69 @@
-.course-announcements {
-  &.index {
-    .announcement {
-      padding-left: 8px;
-
-      .timestamp {
-        color: $gray-light;
-      }
-
-      .content {
-        margin-top: 10px;
-      }
-    }
+.announcement {
```

Comment:
The entire thing should be moved under 
```scss
.course-announcements {
    &.index {
    }
}
```

So that it only applies to announcement index page.